lib/helpers.js

```diff
@@ -42,20 +42,48 @@ const format = (event, options = { ignoreSelection: false }) => {
   const selectedText = editor.getSelectedText();
   const isTransformingFile = options.ignoreSelection || !selectedText;
 
-  const textToTransform = isTransformingFile ? editor.getText() : selectedText;
+  const currentScope = getCurrentScope();
+  const isEmbedded = ['text.html.vue', 'text.html.basic'].includes(currentScope);
+  if (isEmbedded) {
+    const re = /<script\b[^>]*>([\s\S]*?)(?=<\/script>)/gi
+    editor.scan(re, iter => {
+      // only transform first match
+      iter.stop();
```

Comment:
Right. It doesn't work for >1 script blocks, so I put in the guard for now. It could be because the buffer range gets misaligned or something similar. Worth investigating since >1 blocks should probably be supported. Btw, manually formatting two selections doesn't work currently either. Let me know if you have any ideas.